### DISCUSSION TITLE
how to convert back to utf-8

### DISCUSSION THREAD

**Discussion by Goang14** [mpdf/mpdf#1974] [Q&A]
![image](https://github.com/mpdf/mpdf/assets/107413554/00bb5dc0-5e04-435e-baad-87cd4d1ed6e6)
I have a task like this, I want to convert it to the desired Vietnamese text (ĐƠN CAM KẾT)

i am using ctimes in mpdf

**Comment by ssieb** [comment]
That looks like the characters are gone.  I don't think you're going to get it back.  Where did the text come from?

**Reply by Goang14** [reply]
When I use 'dejavusans' it works, but when I switch to 'ctimes' it gives the above error.

**Reply by Goang14** [reply]
- đây là khi tôi sử dụng dejavusans
![image](https://github.com/mpdf/mpdf/assets/107413554/7875d745-e7f2-44a1-837f-e8169ac95fa3)

- đây là khi tôi sử dụng ctimes
![image](https://github.com/mpdf/mpdf/assets/107413554/ee0debe7-39a9-4eb0-a439-4093c4ce63a2)

**Reply by ssieb** [reply]
Then the font just doesn't have those characters.  You'll have to use a font that does.

**Reply by Goang14** [reply]
help me fix this.

**Reply by ssieb** [reply]
Don't use "ctimes".  (I don't even know what font that is.)

**Reply by Goang14** [reply]
it is Times New Roman.
